atlasdb-coordination-impl/src/main/java/com/palantir/atlasdb/coordination/keyvalue/KeyValueServiceCoordinationStore.java

```diff
@@ -172,21 +206,31 @@ public void putValue(long sequenceNumber, ByteString value) {
     }
 
     private static SequenceAndBound deserializeSequenceAndBound(byte[] contents) {
+        return deserializeData(contents, SequenceAndBound.class, "coordination sequence and bound");
+    }
+
+    private static <T> T deserializeData(byte[] data, Class<T> clazz, String safeDescriptionOfItemToDeserialize) {
```

Comment:
This is static, so it can't use the class variable anyway, but should we rename the class var to `valueClass` for clarity?